CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 **IMPORTANT**: scikit-bio is no longer compatible with Python 2. scikit-bio is compatible with Python 3.4 and later.
 
 ### Features
+* Expanded error message in `skbio.io.format.stockholm` to provide better explanation to user. ([#1327](https://github.com/biocore/scikit-bio/issues/1327))
```

Comment:
Can you describe what error message you're expanding, and that it is the Stockholm format reader? There's several types of errors that can be thrown when reading.